addons/questify/scripts/model/quest_start.gd

```diff
@@ -14,3 +14,17 @@ func get_active() -> bool:
 	
 func get_completed() -> bool:
 	return active
+
+
+func serialize() -> Dictionary:
+	return {
+		id = id,
```

Comment:
Just to make it easier to extend/change in the future, let's use `super` here to get the actual implementation from the superclass:

```gdscript
func serialize() -> Dictionary:
	var data = super()
	data.active = active
	return data
	
	
func deserialize(data: Dictionary) -> void:
	super(data)
	active = data.active
```